client/additional-methods-setup/upe-preview-methods-selector/index.js

```diff
@@ -11,26 +11,14 @@ import Wizard from '../wizard/wrapper';
 import WizardTask from '../wizard/task';
 import WizardTaskList from '../wizard/task-list';
 import SetupCompleteTask from './setup-complete-task';
-import useIsUpeEnabled from '../../settings/wcpay-upe-toggle/hook';
 import AddPaymentMethodsTask from './add-payment-methods-task';
 import './index.scss';
 
 const UpePreviewMethodsSelector = () => {
-	const [ isUpeEnabled ] = useIsUpeEnabled();
-
 	return (
 		<Card className="upe-preview-methods-selector">
 			<CardBody>
-				<Wizard
-					defaultActiveTask={
-						isUpeEnabled
-							? 'add-payment-methods'
-							: 'enable-upe-preview'
-					}
-					defaultCompletedTasks={ {
-						'enable-upe-preview': isUpeEnabled,
-					} }
-				>
+				<Wizard defaultActiveTask={ 'add-payment-methods' }>
```

Comment:
⛏️  to be consistent with the other assignments
```suggestion
				<Wizard defaultActiveTask="add-payment-methods">
```

Reply:
Thanks! ceeb2c98bf73206ceaea4a7ad8347d9326777437